DnDCharacterSheet/Program.cs

```diff
@@ -1,22 +1,45 @@
 using Converters;
+using DnDCharacterSheet;
 using DTOs;
 using Factories;
 using Interfaces;
+using Microsoft.EntityFrameworkCore;
 using Middlewares;
 using Models;
+using Repositories;
 using Services;
 using Strategies;
 
 var builder = WebApplication.CreateBuilder(args);
 
+var environment = builder.Configuration["Environment"];
```

Comment:
This is a nice moment to look into the Options Pattern, you can configure different Configuration Objects ( or just one in this case, something like PersistenceConfiguration ) 

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-8.0